### PR TITLE
Multi-select to delete

### PR DESCRIPTION
**Pull request by Hicham010**
I fixed the bug with the ActionMode. Thanks for pointing it out Almightywesley

### CODE REVIEW

**Review by Almightywesley — APPROVED** [review]
No problem next time if tou need help you can just come to me right away

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MemoriesFragment.java [hunk 1]
```diff
@@ -139,14 +139,20 @@ public void onClick(View v) {
         return view;
     }
 
+    @Override
+    public void onPause() {
+        super.onPause();
+        if(actionMode != null){
+        actionMode.finish();}
+    }
+
```
Comment (MrTheGood) [inline app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MemoriesFragment.java:149]: It's clear that you didn't `ctrl+alt+l`. If you have a second, please still do that. If not, that's fine too.

**Review by MrTheGood — APPROVED** [review]
Good on fixing that issue!

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MemoriesFragment.java (hunk 1 above)
Comment (Hicham010) [reply app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MemoriesFragment.java:149]: I ctrl+alt+l-ed

**Pull request merged**